index.html

```diff
@@ -13,5 +13,16 @@
     <img class="title-image" src="assets/sectionTitleImage.svg" alt="image-title"></img>
     <h1 class="title-text">Front End Tools</h1>
   </div>
+  <br>
+  <div class="main-tech-container">
+    <div class="background-tech-container">
+    </div>
+    <div class="data-tech-container">
+      <img class="tech-image" src="assets/react_icon.svg" alt="tech-image"></img>
```

Comment:
Make this tag self-closing `<img class="..." src=" " />`

Reply:
Done